message/tests.py

```diff
@@ -0,0 +1,97 @@
+from django.contrib.auth import get_user_model
+from player.models import Player, BallGame
+from game_event.models import GameEvent
+from message.models import Message
+from django.core.exceptions import ValidationError
+from datetime import datetime
+import pytest
+
+
+@pytest.mark.django_db
+class TestMessageModel:
+
+    @pytest.fixture
+    def user(self):
+        return get_user_model().objects.create_user(
+            username='testuser',
+            password='testpass'
+        )
+
+    @pytest.fixture
+    def saved_player(self, user):
+        player = Player.objects.create(
+            user=user,
+            birth_date='1990-01-01',
+            favorite_ball_game=BallGame.Soccer
+        )
+        return player
+
+    @pytest.fixture
+    def saved_game_event(self):
+        game_event = GameEvent.objects.create()
+        return game_event
+
+    @pytest.fixture
+    def saved_message(self, saved_player, saved_game_event):
+        message = Message.objects.create(
+            user_id=saved_player,
+            game_event_id=saved_game_event,
+            time_sent=datetime(2023, 4, 18, 12, 30, 0),
+            text="test message"
+        )
+        return message
+
+    def test_get_message(self, saved_message):
+        message = Message.objects.get(pk=saved_message.pk)
+        assert message == saved_message
+
+    def test_update_message(self, saved_message):
+        message = saved_message
+        assert message.text != "test message?!"
+        message.text = "test message?!"
+        message.save()
+        updated_message = Message.objects.get(pk=message.pk)
+        assert updated_message.text == "test message?!"
```

Comment:
Making it more readable and maintainable:
```suggestion
    def test_update_message(self, saved_message):
        new_text = "test message?!"
        message = saved_message
        assert message.text != new_text
        message.text = new_text
        message.save()
        updated_message = Message.objects.get(pk=message.pk)
        assert updated_message.text == new_text
```

Reply:
fixed